hugo/layouts/partials/share.html

```diff
@@ -0,0 +1,35 @@
+<details class="share" data-dropdown>
+    <summary class="share__container">
+        <a class="button button--icon button--light-blue share__button">
```

Comment:
what is the reason we have an a inside a summary? The a doesnt link to antyhing: the summary is the thing thats clickable/focusable. Also a without a href is bad practice since it doesn't do anything. If you need antyhing for javascript purposes use a <button> but in this case I think all click events happen on the summary element so I think you can just remove this a and move all classnames to the summary element